### PR TITLE
Provide System.Composition.TypedParts package readme

### PR DESCRIPTION
**Pull request by eNeRGy164**
Fill out details for the `System.Composition.TypedParts` NuGet package readme.

See https://github.com/dotnet/runtime/issues/99358 and https://github.com/dotnet/runtime/pull/106443

### CODE REVIEW

File: src/libraries/System.Composition.TypedParts/src/PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## Additional Documentation
+
+<!-- Links to further documentation. Remove conceptual documentation if not available for the library. -->
+
+* [Managed Extensibility Framework (MEF)](https://learn.microsoft.com/dotnet/framework/mef/)
+
+## Related Packages
+
+<!-- The related packages associated with this package -->
+
```
Comment (ericstj) [inline src/libraries/System.Composition.TypedParts/src/PACKAGE.md:76]: For all of these, should we link to the `System.Composition` meta-package?

**Review by ericstj — APPROVED** [review]
LGTM, @steveharter @buyaa-n

File: src/libraries/System.Composition.TypedParts/src/PACKAGE.md (hunk 1 above)
Comment (eNeRGy164) [reply src/libraries/System.Composition.TypedParts/src/PACKAGE.md:76]: Sure

**Review by steveharter — APPROVED** [review]

**Review by buyaa-n — APPROVED** [review]
Looks great, thank you @eNeRGy164!

**Pull request merged**